scripts/issue/926/register_drippers_2023.py

```diff
@@ -0,0 +1,54 @@
+"""
+ref: https://github.com/smartcontractkit/keeper/blob/master/contracts/UpkeepRegistrationRequests.sol
+"""
+
+from brownie import chain, interface
+
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+
+
+LINK_MANTISSA = 80e18
+UPKEEP_ID_REMBADGER_22Q4 = 142  # https://keepers.chain.link/mainnet/142
+
```

Comment:
great opp to batch cancellations, bvecvx module does not run anymore upkeeps
```suggestion
UPKEEP_ID_BVECVX_MODULE = 14270489937535892760600590423564455802049459622872924518248134412357549386527 # https://automation.chain.link/mainnet/14270489937535892760600590423564455802049459622872924518248134412357549386527
```